modules/javafx.web/src/main/native/Source/WebCore/platform/graphics/java/PathJava.cpp

```diff
@@ -120,9 +120,11 @@ UniqueRef<PathImpl> PathJava::clone() const
 {
     auto platformPathCopy = createEmptyPath();
 
+    RefPtr<RQRef> pathCopy(copyPath(platformPath()));
```

Comment:
The `platformPathCopy` local variable is now unused and can be removed. Better still, you could make the following change and then revert the name change in the `create` method where it is used.

```
    RefPtr<RQRef> platformPathCopy(copyPath(platformPath()));
```